### ISSUE TITLE
Deprecate the Docusaurus config

### ISSUE THREAD

**Issue by slorber** [tsconfig/bases#196]
We now have an internal TS config versioned alongside the rest of the packages: https://github.com/facebook/docusaurus/pull/9050

As discussed previously [here](https://github.com/tsconfig/bases/pull/122), we'd like to deprecate the Docusaurus base config of this repo. Not sure how to do so though 😅

**Comment by orta** [comment]
Makes sense, we can add support for deprecating at npm level to this repo - but there's no (safe) way to deprecate from the typescript side.

 (The unsafe way is to break the build intentionally, but that's a bit rude)
